src/modules/common/components/Page.js

```diff
@@ -38,13 +38,16 @@ type PropType = {
   content: string,
   theme: ThemeType,
   navigation: NavigationScreenProp<*>,
+  navigateToIntegreatUrl: (url: string, cityCode: string, language: string) => void,
   files: FileCacheStateType,
   children?: React.Node,
   language: string,
   cityCode: string,
   lastUpdate: Moment
 }
 
+const HIJACK = /https?:\/\/(cms(-test)?\.integreat-app\.de|web\.integreat-app\.de|integreat\.app)(?!\/[^/]*\/(wp-content|wp-admin|wp-json)\/.*).*/
```

Comment:
Not quite sure if I should leave the cms-test capture.

Reply:
I think it should stay because without it content created in the cms-test would not have internal linking.